README.md

```diff
@@ -48,30 +48,48 @@ Get access to Django CMS, run `docker exec -it core_portal_cms /bin/bash` and th
 python3 manage.py createsuperuser
 ```
 
-In Django CMS admin (i.e. https://cep.test/admin/`), you need to create single page page (using a snippet).
+In Django CMS admin (i.e. https://cep.test/admin/`), you need to create a page  that contains one custom code snippet (to embed the iframe).
 
+#### Creating the Snippet
 
-The iframe snippet has the following markup:
-
+* Login to the `CMS Admin > Snippets`.
+* Add a `New Snippet`.
+* Name it `Data and Analysis`.
+* Copy & paste the following markdown code into the body field:
 ```
-<body style="margin:0px;padding:0px;overflow:hidden">
-    <div style="position:relative;width:100vw;height:100vh;">
-        <iframe src="https://cep.test/protx/dash/" frameborder="0" allowfullscreen style="position:absolute;top:0;left:0;width:100%;height:100%;" width="100%" height="100%"></iframe>
+<div style="margin:0; padding:0;">
+    <div style="display:flex; flex-direction:column; min-height:100vh;">
+        <iframe style="border:none; flex-grow:1;" src="https://cep.test/protx/dash/analytics"></iframe>
```

Comment:
```suggestion
        <iframe style="border:none; flex-grow:1;" src="https://cep.test/protx/dash/"></iframe>
```
Recent changes make this path to be just dash/

Reply:
Werd!